api/src/main/java/org/pmiops/workbench/utils/mappers/WorkspaceMapper.java

```diff
@@ -35,20 +35,6 @@
       WorkspaceResourceMapper.class,
     })
 public interface WorkspaceMapper {
-
```

Comment:
here is the main change - remove the version which doesn't include Featured Workspaces.

This is safe to use with either FF setting.  The underlying DB rows can only have Published set when the FF is false, and Featured Category only when the FF is true.